openfecli/commands/quickrun.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import click
+from openfecli import OFECommandPlugin
+import json
+
+
+@click.command(
+    'quickrun',
+    short_help="Run a given transformation, saved as a JSON file"
+)
+@click.argument('transformation', type=click.File(mode='r'),
+                required=True)
+def quickrun(transformation):
+    """Run the transformation (edge) in the given JSON file in serial.
+
+    To save a transformation as JSON, use the following Python recipe (after
+    creating the transformation):
+
+    \b
+        import json
+        with open(filename, 'w') as f:
+            json.dump(transformation.to_dict(), f)
+    """
+    import gufe
+    from gufe.protocols.protocoldag import execute
+    print("Loading file...")
+    dct = json.load(transformation)
+    trans = gufe.Transformation.from_dict(dct)
+    print("Planning the campaign...")
```

Comment:
I suspect these are going to be lost in the amazing amounts of noise our protocol generates. This isn't a problem for now, but we should think about how we deal with our stdout / stderr streams.

Reply:
I've abstracted this into `openfecli.utils.write`, so we can easily change the details later. FWIW, this is only here because I was curious as to what was taking so long with the dummy protocol (`create` was slow... I think this got fixed by some updates Dotson made).